components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -1007,7 +1007,7 @@
 <config_AM_transectTransport_transect_group>'all'</config_AM_transectTransport_transect_group>
 
 <!-- AM_eddyProductVariables -->
-<config_AM_eddyProductVariables_enable>.true.</config_AM_eddyProductVariables_enable>
+<config_AM_eddyProductVariables_enable>.false.</config_AM_eddyProductVariables_enable>
```

Comment:
I think this is okay for now but we might want it off for some meshes and on of others in the future.

Reply:
I think we can put this on the back burner until we develop the RRS6to18 compset for v3, then we'll want to have the mesh specific enabling.